### PR TITLE
[Design2-68] Share Project Modal (Dance Party Page) - Use DSCO Buttons / Align bottom buttons

### PR DESCRIPTION
**Pull request by levadadenys**
### [Design2-68] Share Project Modal (Dance Party Page) - Use DSCO Buttons / Align bottom buttons

* use DSCO Buttons in ShareAllowedDialog.jsx (Share Project Dialog)

Before:
Знімок екрана 2024-06-03 о 16 00 07
Знімок екрана 2024-06-03 о 18 00 57

After:
Знімок екрана 2024-06-03 о 16 00 04
Знімок екрана 2024-06-03 о 17 53 25

## Links
- jira ticket: [Design2-68](https://codedotorg.atlassian.net/browse/DESIGN2-68)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/components/ShareAllowedDialog.jsx [hunk 1]
```diff
@@ -372,14 +372,10 @@ class ShareAllowedDialog extends React.Component {
...
                   <div>
                     <Button
-                      color={Button.ButtonColor.brandSecondaryDefault}
+                      color={buttonColors.purple}
+                      type="primary"
                       id="sharing-dialog-copy-button"
-                      icon="clipboard"
-                      style={{
-                        ...styles.button,
-                        ...styles.copyButton,
```
Comment (bencodeorg) [inline apps/src/code-studio/components/ShareAllowedDialog.jsx:380]: Can we remove this styling now?

File: apps/src/code-studio/components/ShareAllowedDialog.jsx [hunk 2]
```diff
@@ -396,26 +392,24 @@ class ShareAllowedDialog extends React.Component {
                 </div>
                 <div className="social-buttons" style={{marginTop: 12}}>
                   <Button
-                    color={Button.ButtonColor.neutralDark}
-                    id="sharing-phone"
```
Comment (bencodeorg) [inline apps/src/code-studio/components/ShareAllowedDialog.jsx:400]: Any idea what this ID was doing? [This looks maybe relevant](https://github.com/code-dot-org/code-dot-org/blob/fc1a5df28458eb97241a6e953559609a2e27a75f/apps/src/feedback.js#L995)?

File: apps/src/code-studio/components/ShareAllowedDialog.jsx (hunk 1 above)
Comment (levadadenys) [reply apps/src/code-studio/components/ShareAllowedDialog.jsx:380]: Thanks for the catch! We still do need `styles.button`, but `.copyButton` indeed can be removed. (making this update now)

File: apps/src/code-studio/components/ShareAllowedDialog.jsx (hunk 2 above)
Comment (levadadenys) [reply apps/src/code-studio/components/ShareAllowedDialog.jsx:400]: Yep, also might be the reason of faliing ui test, thanks a lot for spotting!

**Review by bencodeorg — APPROVED** [review]
Looks so much better, nice work!

**Pull request merged**